harness/byteConversionValues.js

```diff
@@ -483,7 +633,22 @@ var byteConversionValues = {
       -4294967296, // -4294967296
       Infinity,    // Infinity
       -Infinity,   // -Infinity
-      0
+      0,           // 0
+      2049,                    // 2049
+      2051,                    // 2051
+      0.00006103515625,        // 0.00006103515625
+      0.00006097555160522461,  // 0.00006097555160522461
+      5.960464477539063e-8,    // 5.960464477539063e-8
+      2.9802322387695312e-8,   // 2.9802322387695312e-8
```

Comment:
the eslint "no precision loss" rule complains about this value, is that a concern?